clinica/pipelines/t1_linear/t1_linear_pipeline.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
+import clinica.pipelines.engine as cpe
+
+
+class T1Linear(cpe.Pipeline):
+    """T1 Linear SHORT DESCRIPTION.
```

Comment:
I propose to use the tagline of the future Wiki page :)
```suggestion
    """T1 Linear - Affine registration of T1w images to standard space.
```